code/mob/living/carbon/human/procs/update_icon.dm

```diff
@@ -866,7 +866,7 @@
 		src.image_cust_three?.layer = src.bioHolder.mobAppearance.customization_third.default_layer
 
 
-var/list/update_body_limbs = list("r_arm" = "stump_arm_right", "l_arm" = "stump_arm_left", "r_leg" = "stump_leg_right", "l_leg" = "stump_leg_left")
+var/list/update_body_limbs = list("r_leg" = "stump_leg_right", "l_leg" = "stump_leg_left", "r_arm" = "stump_arm_right", "l_arm" = "stump_arm_left")
```

Comment:
Why the re-order?

Reply:
It was because Eldritch arms are supposed to layer over legs, and with the original order it was making the arm part that appears over the leg appear under.